README.md

```diff
@@ -1,2 +1,127 @@
+for the [Solana Program Library](https://github.com/solana-labs/solana-program-library) (SPL) repository or the 
+[Solana](https://github.com/solana-labs/solana) monorepo.
+
+### Level 2 - Write Access
+
+Requirements: One voucher from anyone with level 3 access or two vouchers from
+anyone from level 2. Applicants must have level 1 access to apply for level 2.
+
+Contributors with Write access will have the associated [write Github access
+policy](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role)
+for the [Solana Program Library](https://github.com/solana-labs/solana-program-library) (SPL) repository or the 
+[Solana](https://github.com/solana-labs/solana) monorepo.
+
+Most notably these permissions allow the contributor to approve and merge pull
+requests.
+
+### Level 3 - Maintain Access
+
+Requirements: One voucher from anyone with level 3 access.
+
+Contributors will have the associated [maintain Github access
+policy](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role)
+for the [Solana Program Library](https://github.com/solana-labs/solana-program-library) (SPL) repository or the 
+[Solana](https://github.com/solana-labs/solana) monorepo.
+
+Users with Maintain access are responsible for managing the the repositories and
+granting user access based on the above requirements.
+
+### Vouching Process 
+
+To receive a Contributor status and the respective level of access as defined in the Contributor Access Policy, follow these steps:
+
+1. Open an issue on the [Solana Program Library](https://github.com/solana-labs/solana-program-library) (SPL) repository 
+with a title in the following format: "Request Access (Level [X])
+for [Username]". Add information that can help identify you such as your 
+Discord ID in the issue.
+2. Gather your vouchers to add a comment on the issue expressing their support.
+3. Once the issue has received enough support, notify the users with Maintain
+access by adding a comment tagging
+[`@contributors`](https://github.com/orgs/solana-labs/teams/contributors)
+and you will be granted the requested level of access.
+
+To receive a Core Contributor status and the respective level of access as defined in the Contributor Access Policy, follow these steps:
+
+1. Open an issue on the [Solana](https://github.com/solana-labs/solana) monorepo
+with a title in the following format: "Request Access (Level [X])
+for [Username]". Add information that can help identify you such as your 
+Discord ID in the issue.
+2. Gather your vouchers to add a comment on the issue expressing their support.
+3. Once the issue has received enough support, notify the users with Maintain
+access by adding a comment tagging
+[`@core-contributors`](https://github.com/orgs/solana-labs/teams/core-contributors)
```

Comment:
Can probably use a more narrowly scoped group here. I feel more than 1 person and less than 4 people should be responsible for managing ACLs